blocks/quote/quote.js

```diff
@@ -0,0 +1,33 @@
+import {
+  div,
+} from '../../scripts/dom-builder.js';
+
+export default function decorate(block) {
+  const quoteTextBlock = block.querySelector(':scope > div:first-of-type');
+  const quoteSourceBlock = block.querySelector(':scope > div:nth-of-type(2)');
+  block.innerHTML = '';
+
+  const quoteTextContainer = div({ class: 'block content' });
+  quoteTextContainer.append(...quoteTextBlock.childNodes);
+
+  const quoteSourceContainer = div({ class: 'block content' });
+  quoteSourceContainer.append(...quoteSourceBlock.childNodes);
+
+  // quoteText row
+  const qtRow = div(
+    { class: 'row' },
+    div({ class: 'block qmcolumn opening-quotationmark' }),
+    quoteTextContainer,
+    div({ class: 'block qmcolumn closing-quotationmark' }),
+  );
+  block.append(qtRow);
+
+  // quoteSource row
+  const qsRow = div(
```

Comment:
Why do we need this complex doc structure with 3 children and two of there are just placeholders?

Reply:
Use case: you have a long quote text and a long quote source. 
The containing box of the text in the two rows must be "synchronized", i.e. long quote text and quote source have to wrap at the same x-pos (see https://sap.frontify.com/document/223142#/-/quote). The easiest way to achieve this (to me) was to repeat the same row structure in both rows to ensure the same styling applies to both rows

Reply:
As @larsauffarth wrote only CSS should be enough here. Also for the quote text the extra columns should not be needed and `::after` `::before` can be used.

Reply:
Removed the placeholders in 2nd row